config/router.config.js

```diff
@@ -74,6 +74,11 @@ module.exports = [
             icon: 'dashboard',
             component: './Overview',
           },
+          {
+            path: '/private/analysis',
```

Comment:
I am not sure about these `/private` routes.  Don't think we want to provide routes based on their access, but on their function, and then determine the behavior based on access.